### ISSUE TITLE
choose tab: data availability by sensordata

### ISSUE THREAD

**Issue by elma13** [rivm-syso/Analyse-Together#391]
Get the data availability from the sensor data in stead of the data time ranges in the cache table. The data itself is maybe more robust than the cache table. NB check the performance in time.

**Comment by elma13** [comment]
- [ ]  check if there are sensors known in the cache dbs
- [ ] if there are sensors known, get_measurements_cleaned
- [ ] if there is data, count number sensors per day
- [ ] show those in the same calender + map as on start page! 
- [ ] Show both buttons (use data, download data), let the user decide. They know how many sensors to expect.

**Comment by elma13** [comment]
Store the data in separated reactivevalues till the user press the button "use data", then store the data in sensor_data. In this way only once the get_measurements_cleaned is used/

**Comment by elma13** [comment]
> Store the data in separated reactivevalues till the user press the button "use data", then store the data in sensor_data. In this way only once the get_measurements_cleaned is used/

Uhm, also the time period, and station info is needed to be updated. For now keep it in the "use data" observeEvent.

**Comment by elma13** [comment]
> * [x]   check if there are sensors known in the cache dbs
> * [x]  if there are sensors known, get_measurements_cleaned
> * [ ]  if there is data, count number sensors per day
> * [ ]  show those in the same calender + map as on start page!
> * [x]  Show both buttons (use data, download data), let the user decide. They know how many sensors to expect.

For the map, the station data needs to be available

**Comment by elma13** [comment]
Do the part of the map and store to get the data in the START-page only once the data in a new issue.
